docs/reference/pipeline-yaml-reference.md

```diff
@@ -1548,6 +1548,50 @@ blocks:
 In this case the name of the file that was saved in a `secret` is `file.txt`
 and is located at the home directory of the VM.
 
+## after_pipeline
+
+The `after_pipeline` property is used for defining a set of jobs that will
+execute when the pipeline is finished. The `after_pipeline` property is most
+commonly used for sending notifications, collecting test results, and submitting
+metrics.
+
+For example, to submit pipeline duration metrics and to publish test results, you
+would define the after pipeline with the following YAML snippet:
+
+``` yaml
+after_pipeline:
+  task:
+    jobs:
+      - name: Submit Metrics
+        commands:
+          - echo "ci.duration:$SEMAPHORE_PIPELINE_TOTAL_DURATION|ms" | nc -w 3 -u statsd.example.com
```

Comment:
According to the env vars section, this value is in seconds and here it is being sent as milliseconds.
Even though this is just an example, someone can take it literally and end up with wrong values.